.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Fri Sep 16 12:01:40 IST 2022
+gradle.version=6.1.1
```

Comment:
these files Should not be committed. Please remove the binary and temp files from commit/merge